pages/common/git-imerge.md

```diff
@@ -0,0 +1,24 @@
+# git-imerge
+
+> Perform a merge or rebase between two git branches incrementally.
+> Conflicts between branches are tracked down to pairs of individual commits, to simplify conflict resolution.
+
+- Start imerge-based rebase (checkout the branch to be rebased, first):
+
+`git imerge rebase {{branch_to_rebase_into}}`
```

Comment:
I suggest changing the argument name to `branch_to_rebase_onto`, since your branch will end up on top of the other branch; the other branch is not changed.